sources/updater/updog/src/main.rs

```diff
@@ -479,12 +514,12 @@ fn main_inner() -> Result<()> {
                 &manifest,
                 &current_release.version_id,
                 &variant,
-                arguments.force_version,
+                get_target_version(&config, &arguments.force_version)?,
             ) {
-                if u.update_ready(config.seed) || arguments.ignore_waves {
+                if u.update_ready(config.seed) || (arguments.ignore_waves || config.ignore_waves) {
```

Comment:
nit: if we flip the order, we can avoid the function call if we know we're ignoring waves.